src/hierarchy/Birds.ts

```diff
@@ -0,0 +1,26 @@
+import { ICanFly } from "../interfaces/ICanFly"
+import { ICanTalk } from "../interfaces/ICanTalk"
+import Animal, { AnimalCandidate } from "./Animal"
+
+export default abstract class Birds extends Animal implements ICanFly, ICanTalk {
+    fly(): string {
+        return `${this.name} is flying`
+    }
+    talk(): string {
+        return `${this.name} is talking`
+    }
+
```

Comment:
All the birds can also walk from what i can see maybe we add that interface as well

Reply:
there is no need, , super class Animal has it implemented, so all animals can walk